event_listener.py

```diff
@@ -103,11 +102,8 @@ def on_init(self,views):
 				conversation = view.settings().get("conversation_id")
 				if conversation:
 					on_close = lambda x:copilot.render_conversation(conversation)
-					sublime.set_timeout(lambda: view.close(on_close),5000)# Wait some sec until the conversations is loaded
+					sublime.set_timeout(lambda: view.close(on_close),20000)# Wait some sec until the conversations is loaded
```

Comment:
lot of time to be waiting here, any reason why we need to have this here? typically bad practice to have arbitrary delays as this will be dependant on a number of different factors

Reply:
yes we can do better will need to create a github issue 
so basically we want to the websocket to open I can create a github issue to make the command is not enabled until the websocket opened we can leave it for now like that thank you

Reply:
https://github.com/pieces-app/plugin_sublime/issues/120

Reply:
@Bishoy-at-pieces if we are waiting for the websocket to open it's probably better to hook into the websocket.onopen callback

probably need to use some form of asynchronous code for this case.